src/api/custom-fetch.ts

```diff
@@ -1,12 +1,19 @@
 import { config } from "../config.js";
+import packageJson from "../../package.json" assert { type: "json" };
 
 export const customFetch = async (url: string, options?: RequestInit) => {
   const headers: HeadersInit = {
     "X-Redmine-API-Key": config.redmineApiKey,
+    "Accept": "application/json",
```

Comment:
Since there are cases like file downloads, I cannot always expect a JSON return from every API.
I think it would be best to remove Accept (and let Redmine handle it), what do you think?